src/config/dynamic-theme-fixes.config

```diff
@@ -1070,8 +1070,8 @@ INVERT
 aprs.fi
 
 INVERT
-.gm-style > :first-child
-.gm-style-iw-a
+img[src^="https://maps.googleapis.com/maps/"]
```

Comment:
Does this fixes any existing issues?

Reply:
It does. Node icons are displayed inverted on the map, but not on the pop-up:
![image](https://user-images.githubusercontent.com/17483811/125243969-0abaaf80-e2ef-11eb-9b0f-e18a3588942b.png)
After the commit, both icon types are non-inverted.

Reply:
Both should be inverted. The non-inverted version doesn't look that good in dark-mode environment.